testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -1500,4 +1500,71 @@ public boolean isSupportedInTrino()
         CREATE_TABLE_AS_SELECT,
         CREATE_TABLE_WITH_NO_DATA_AND_INSERT,
     }
+
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS}, dataProvider = "storageFormats")
+    public void testSparkReadsTrinoTableAfterCleaningUp(StorageFormat storageFormat)
```

Comment:
nit: `AfterCleaningUp` -> `AfterExpiringSnapshots` (more explicit)

Reply:
That is not correct. By cleaning up I mean executing both EXPIRE_SNAPSHOT and DELETE_ORPHAN_FILE

Reply:
If you have better naming let me know. Concatenating both seemed way to long